capsules/extra/src/bus.rs

```diff
@@ -315,6 +329,8 @@ impl<'a, I: I2CDevice> Bus<'a> for I2CMasterBus<'a, I> {
         // endianess does not matter as the buffer is sent as is
         let bytes = data_width.width_in_bytes();
         self.len.set(len * bytes);
+
+        //TODO: might crash
```

Comment:
Why might this code crash?

Reply:
I left the marking there because the code is untested and used by other parts of tock. I left it as a reminder to check code integration.

Reply:
The comment was removed.